extensions/sql-database-projects/src/models/project.ts

```diff
@@ -505,6 +513,34 @@ export class Project implements ISqlProject {
 		return noneRemoveScripts;
 	}
 
+	/**
+	 *
+	 * @returns all the publish profiles (ending with *.publish.xml) specified as <None Include="file.publish.xml" /> in the sqlproj
+	 */
+	private readPublishProfiles(): FileProjectEntry[] {
+		const publishProfiles: FileProjectEntry[] = [];
+
+		for (let ig = 0; ig < this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.ItemGroup).length; ig++) {
+			const itemGroup = this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.ItemGroup)[ig];
+
+			// find all publish profile scripts to include
+			try {
+				const noneItems = itemGroup.getElementsByTagName(constants.None);
```

Comment:
For SDK-style projects, I'm wondering if publish profiles need to be added as `None` items or if they can just be glob included. I did a quick check of what C# SDK-style projects do, and when I tried adding an xml file to the project it didn't add it to the .csproj

Reply:
My bad I was of the opinion that SDK style sql projects was only supposed to glob sql files, and not others. I will look into updating this.

Reply:
hmm yeah that would make sense to only show by default what's included in the build, but wasn't sure if the behavior here should be the same as VS. Like when I dragged a .sql file and an .xml file into my test C# project, they showed up in the project tree without anything being added in the .csproj